src/AppBundle/Manager/SubscriptionManager.php

```diff
@@ -93,14 +120,31 @@ public function saveSubscription(Subscription $subscription)
     {
         $this->em->persist($subscription);
         $this->em->flush($subscription);
+
+        $this->dispatcher->dispatch(
+            SubscriptionEvents::POST_WRITE,
+            new SubscriptionEvent($subscription->getId(), $subscription)
+        );
     }
 
     /**
      * @param Subscription $subscription
+     * @param bool $dispatch
      */
-    public function updateSubscription(Subscription $subscription)
-    {
+    public function updateSubscription(
+        Subscription $subscription,
+        $dispatch = true
```

Comment:
![shushing is absolutely normal](https://media.giphy.com/media/DcJgKENAotVYY/giphy.gif)

You're right Tjeerd, this should be cleaned up.